src/config.rs

```diff
@@ -23,6 +23,14 @@ pub struct Config {
     )]
     /// The logging level of the application
     log_level: LevelFilter,
+
```

Comment:
Let's derive the getter via the `getset` crate:
```suggestion
    #[get = "pub"]
```

Reply:
I tried that first and got into a fight with the borrow checker :) I will reattempt that.

Reply:
Did you specify `get_copy`? For the log_level for example we use `get_copy`, because `LevelFilter` implementy `Copy` (small overhead when copying the type).

For larger types we have to use `get`, because this will result in returning a reference (`&str`) in our case.

Reply:
done